alerts/system_alerts.libsonnet

```diff
@@ -65,14 +65,14 @@
           {
             alert: 'KubeletTooManyPods',
             expr: |||
-              kubelet_running_pod_count{%(kubeletSelector)s} > %(kubeletTooManyPods)s
+              kubelet_running_pod_count{%(kubeletSelector)s} > %(kubeletPodLimit)s * 0.9
```

Comment:
Maybe `%(kubeletPodLimit)s / 1.1` to get 100 instead of 99.

Reply:
sounds fine to me. the point was to abstract away from the specific number of `110`. if `110` is changed in the jsonnet config, then `1.1` isn't really a meaningful number, whereas `90%` would still be.

Reply:
Ah, I see. Let's stick with `.9` then.